Make it clear the flex: 1 is required (#18353)
expo/expo: docs/pages/versions/unversioned/react-native/safeareaview.md

## Example

> To use, wrap your top level view with a `SafeAreaView` with a `flex: 1` style applied to it, so it fills all of the available space. You may also want to use a background color that matches your application's design.

Reviewer: ```suggestion
To use, wrap your top-level view with a `SafeAreaView`. To fill all of the available space, add `flex: 1` as a style to it. You may also want to use a background color that matches your application's design.
```

> To use, wrap your top level view with a `SafeAreaView` with a `flex: 1` style applied to it. You may also want to use a background color that matches your application's design.

Reviewer 2: ```suggestion
To use, wrap your top level view with a `SafeAreaView` with a `flex: 1` style applied to it, so it fills all of the available space. You may also want to use a background color that matches your application's design.
```

Let's expand already existing sentence, instead of adding a new note.

> **Note: The `flex: 1` is required so the `SafeAreaView` will [fill all the available space](https://reactnative.dev/docs/height-and-width#flex-dimensions).**
> 

Reviewer 2: ```suggestion
```

Remove the note.

const styles = StyleSheet.create({
  container: {

>     * @info Remember to set correct style, so the SafeAreaView fills the entire available space.*/flex: 1,/* @end */

Reviewer: Not sure what "Remember to set correct style," means here.

I think, to put the emphasis on the `flex: 1` it can could be:
```suggestion
    /* @info Add this so that the SafeAreaView fills the entire available space.*/flex: 1,/* @end */
```

Also, the is a missing `/` in front of `@info` tag

const styles = StyleSheet.create({
  container: {

>     flex: 1, // required, so the SafeAreaView has a height greater than zero

Reviewer 2: ```suggestion
    * @info Remember to set correct style, so the SafeAreaView fills the entire available space.*/flex: 1,/* @end */
```

Let's use highlight and tooltip instead of plain comment.
